src/Workspaces/Core/Portable/Storage/SQLite/Interop/SqlConnection.cs

```diff
@@ -196,13 +199,15 @@ public Stream ReadBlob(string dataTableName, string dataColumnName, long rowId)
             // the one the BLOB handle is open on. Calls to sqlite3_blob_read() and 
             // sqlite3_blob_write() for an expired BLOB handle fail with a return code of
             // SQLITE_ABORT.
-            Stream stream = null;
-            RunInTransaction(() =>
+            return RunInTransaction((ValueTuple<string, string, long, SqlConnection> tuple) =>
```

Comment:
why not `((string dataTableName, string dataColumnName, long rowId, SqlConnection connection) tuple) =>`?

Reply:
given inference, you might even be able to write it as:

`return RunIntransaction(((dataTableName, dataColumnName, rowId, connection) tuple) => ...`